app/browser/api/ledger.js

```diff
@@ -1069,7 +1069,14 @@ const backupKeys = (state, backupAction) => {
         if (backupAction === 'print') {
           webContents.print({silent: false, printBackground: false})
         } else {
-          webContents.downloadURL(fileUrl(filePath), true)
+          const dialog = electron.dialog
```

Comment:
Can we just move this out of the tab create and do if check. If `backupAction` `print` create tab else open dialog?

Reply:
yeah, we must move it out.